client/src/pages/Login.js

```diff
@@ -144,6 +150,19 @@ const Login = ({ isLoginForm }) => {
             style={StyleInput}
           />
         </InputWrapper>
+        {!isLoginForm && (
+          <InputWrapper>
```

Comment:
I think there's also a special password input component but it was broken for some reason, we probably still have to look at that and finish some other details of this signup &Y login page (like email validation etc.)